### PR TITLE
Compass Calibration: Symptoms and caveats

### PR DESCRIPTION
**Pull request by hamishwillee**
@bkueng This attempts to improve calibration by explaining when you need to calibrate or recalibrate (symptoms), and also a few common gotchas for choosing the wrong location.

@LorenzMeier FYI.

### CODE REVIEW

File: en/config/compass.md [hunk 1]
```diff
@@ -3,12 +3,21 @@
...
+> **Note** If you are using an external magnetometer/compass (e.g. a compass integrated into a GPS module) make sure you mount the external compass on your vehicle properly and connect it to the autopilot hardware.
+  Instructions for connecting your GPS+compass can be found in [Basic Assembly](../assembly/README.md) for your specific autopilot hardware.
+  Once connected, *QGroundControl* will automatically detect the external magnetometer.
+
+<span></span>
+> **Tip** You will need to calibrate your compass on first use, and you may need to recalibrate it if the vehicles is ever exposed to a very strong magnetic field, or if it is used in an area with abnormal magnetic characteristics.
+  Indications of a poor compass calibration include multicopter circling during hover, or veering off-path when attempting to fly straight.
```
Comment (bkueng) [inline en/config/compass.md]: Maybe you want to mention 'toilet bowl'?

Comment (hamishwillee) [reply en/config/compass.md]: Isn't toilet bowl circling during hover? "Definitions" on the Internet range from "rotate in place during hover" to "circling during hover". I would have thought a toilet was an inward spiral leading to being sucked to ground :-)

Comment (hamishwillee) [reply en/config/compass.md]: @bkueng ^^^ Can you please confirm the correct definition of toilet bowling?

Comment (bkueng) [reply en/config/compass.md]: It is starting to circling, starting off small usually, and getting larger until you have a fly-away. So you can say it's a spiral, but the altitude is usually constant.

Comment (hamishwillee) [reply en/config/compass.md]: Thanks very much. Integrated.

**Pull request merged**